include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -482,11 +482,18 @@ struct from_json_fn
 };
 }  // namespace detail
 
+#ifndef JSON_HAS_CPP_17
 /// namespace to hold default `from_json` function
 /// to see why this is required:
 /// http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4381.html
 namespace // NOLINT(cert-dcl59-cpp,fuchsia-header-anon-namespaces,google-build-namespaces)
 {
+#endif
+#ifdef JSON_HAS_CPP_17
```

Comment:
Should this `#endif/#ifdef` be an `#else`?

Reply:
No. As intended.

Reply:
I have to push a small cosmetic fix anyway, so I'm making this change as well.